js/tic_tac_toe.js

```diff
@@ -1,2 +1,131 @@
-// Your code here
-// Use the jQuery
+var $lFirst, $lSecond, $lThird, $rFirst, $rSecond, $rThird, $h1, $star, $circle, $toggle;
+var count, blue_rows, red_rows, blue_cols, red_cols;
+
+$(document).ready(function(){
+  TicTacToe.init();
+  $circle.on('click', TicTacToe.move);
+  $toggle.on('click', TicTacToe.player)
+});
+
+var TicTacToe = (function(){
+
+  var init = function(){
+    $circle = $('.box_cell');
+    $lFirst = $('#box_0_0');
+    $center = $('#box_1_1');
+    $lThird = $('#box_2_2');
+    $rFirst = $('#box_2_0');
+    $rThird = $('#box_0_2');
+    $h1 = $('h1');
+    $star = $('*');
+    $toggle = $('#toggle');
+    count = 0;
+    blue_rows = [];
+    red_rows = [];
+    blue_cols = [];
+    red_cols = [];
+  };
+
+  var _goBlue = function(){
```

Comment:
I think you could make _goBlue and _goRed into one function that's a little more DRY. But these certainly aren't wrong :)